### DISCUSSION TITLE
Mac OS: Constantly looking for folder paths on start up

### DISCUSSION THREAD

**Discussion by lazysod** [iterate-ch/cyberduck#16383] [Q&A]
Every time I start up Cyberduck i keep having to navigate to the following folder path?

```
/Applications/Cyberduck.app/Contents/_MASReceipt
```

I've even gone as far as removing and re installing and nothing seems to work. Anyone else having this issue?

**Comment by dkocher** [comment]
Duplicate for #16367.
